src/directive/translate-language.js

```diff
@@ -54,13 +54,17 @@ function translateLanguageDirective() {
     scope: true,
     compile: function () {
       return function linkFn(scope, iElement, iAttrs) {
+
         iAttrs.$observe('translateLanguage', function (newTranslateLanguage) {
           scope.translateLanguage = newTranslateLanguage;
         });
+
+        scope.$watch('translateLanguage', function(){
+          scope.$broadcast('translateLanguageChanged');
```

Comment:
The purpose of notifying child scopes is to make sure that all child nodes will be updated accordingly. I have added two unit tests to the existing stack that are testing the following:
** it('should not change parent scope language', ...) **- the current translateLanguage change should not affect translations in the parent scopes
** it('should not be changed by parent scope language'** - preserve language preference below the current scope